softdev.bib

```diff
@@ -12,6 +12,19 @@
 
 For 'month' fields, please use the abbreviated 3 letter notation.
 
+@article{sison95simultaneous,
+  title={Simultaneous confidence intervals and sample size determination for multinomial proportions},
+  author={Sison, Cristina P and Glaz, Joseph},
```

Comment:
That should probably be either "Cristina P." (with a full stop) or just "Cristina"?